### PR TITLE
Jira query add description

### PR DESCRIPTION
**Pull request by thefrieddan1**
## Contributing to Cortex XSOAR Content
Make sure to register your contribution by filling the [contribution registration form](https://forms.gle/XDfxU4E61ZwEESSMA)

**The Pull Request will be reviewed only after the contribution registration form is filled.**

## Status
- [x] In Progress
- [ ] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
fixes: https://jira-hq.paloaltonetworks.local/browse/CIAC-2149

## Description
Add field description to output of the command jira-issue-query. in order to support the JIRA-AHA mirror
## Screenshots
Paste here any images that will help the reviewer

## Does it break backward compatibility?
   - [ ] Yes
       - Further details:
   - [ ] No

## Must have
- [ ] Tests
- [ ] Documentation

### CODE REVIEW

File: Packs/Jira/Integrations/JiraV2/JiraV2.yml [hunk 1]
```diff
@@ -171,6 +171,9 @@ script:
     - contextPath: Ticket.Summary
       description: The ticket summary.
       type: Unknown
+    - contextPath: Ticket.Description
+      description: The ticket project description.
```
Comment (JasBeilin) [inline Packs/Jira/Integrations/JiraV2/JiraV2.yml:175]:
```suggestion
      description: The ticket description.
```

**Review by JasBeilin — COMMENTED** [review]
Looks good, make sure to add release notes.

File: Packs/Jira/ReleaseNotes/2_0_17.md [hunk 2]
```diff
@@ -0,0 +1,5 @@
+
+#### Integrations
+##### Atlassian Jira v2
+- Updated the Docker image to: *demisto/oauthlib:1.0.0.36344*.
+- Added a field description to command jira-issue-query output.
```
Comment (tkatzir) [inline Packs/Jira/ReleaseNotes/2_0_17.md:5]:
```suggestion
- Added the field **description** to command **jira-issue-query output**.
```

File: Packs/Jira/Integrations/JiraV2/JiraV2.py [hunk 3]
```diff
@@ -6,7 +6,7 @@
 from CommonServerPython import *
 
 # Disable insecure warnings
-requests.packages.urllib3.disable_warnings()
+requests.packages.urllib3.disable_warnings()   # type: ignore[attr-defined]
```
Comment (JasBeilin) [inline Packs/Jira/Integrations/JiraV2/JiraV2.py:9]: change to urllib3

**Review by JasBeilin — CHANGES_REQUESTED** [review]

File: Packs/Jira/ReleaseNotes/2_0_17.md (hunk 2 above)
Comment (thefrieddan1) [reply Packs/Jira/ReleaseNotes/2_0_17.md]: done.

File: Packs/Jira/Integrations/JiraV2/JiraV2.py (hunk 3 above)
Comment (thefrieddan1) [reply Packs/Jira/Integrations/JiraV2/JiraV2.py]: done.

File: Packs/Jira/ReleaseNotes/2_0_17.md [hunk 4]
```diff
@@ -0,0 +1,5 @@
+
+#### Integrations
+##### Atlassian Jira v2
+- Updated the Docker image to: *demisto/oauthlib:1.0.0.36344*.
+- Added the field **description** to the **jira-issue-query** command output.
```
Comment (ShahafBenYakir) [inline Packs/Jira/ReleaseNotes/2_0_17.md:5]:
```suggestion
- Added the field **description** to the ***jira-issue-query*** command output.
```

**Review by ShahafBenYakir — APPROVED** [review]

**Review by JasBeilin — APPROVED** [review]

**Pull request merged**